habitat-lab/habitat/config/__init__.py

```diff
@@ -4,123 +4,8 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-r"""Habitat Lab Configuration
```

Comment:
Was this comment used anywhere? Maybe it is used to generate documentation. Do you know @mathfac ?

Reply:
I haven't found it in the documentation.